envi/__init__.py

```diff
@@ -1186,21 +1186,23 @@ def executeCall(self, emu, va, args=None, ra=None):
         self.setupCall(emu, args=args, ra=ra)
         emu.setProgramCounter(va)
 
-    def execCallReturn(self, emu, value, argc):
+    def execCallReturn(self, emu, value, argc, apiReturnsVoid=False):
```

Comment:
this is not the way to do this.  
this is the calling convention object.  theoretically, it *knows* whether the function should return Void.